renderer/components/Channels/ChannelCardListItem.js

```diff
@@ -29,8 +32,9 @@ const ChannelCardListItem = React.memo(
       active,
     } = channel
     const opacity = active ? 1 : 0.3
+
     return (
-      <Card {...rest}>
+      <Card>
```

Comment:
Why remove `...rest` from here? This makes the component less configurable by the caller

Reply:
I've removed it until it's actually needed. We can't blindly pass anything there because previously it was causing superfluous renders. This is a narrowly scoped component with perf focused code